recipes/protoc/3.9.x/conanfile.py

```diff
@@ -35,13 +36,17 @@ def requirements(self):
         self.requires.add("protobuf/3.9.1", private=True)
 
     def _configure_cmake(self):
-        cmake = CMake(self)
-        cmake.definitions["protobuf_BUILD_TESTS"] = False
-        cmake.definitions["protobuf_WITH_ZLIB"] = False
+        if self._cmake:
+            return self._cmake
+        self.settings._data["arch"] = self.settings._data["arch_build"].copy()
```

Comment:
@madebr we can't set any internal attribute. As you can see, _data is protected.

Reply:
Indeed, oops.